charts/osm-arc/Chart.yaml

```diff
@@ -15,12 +15,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0
+version: 1.0.0.1
```

Comment:
This format is not [valid semver](https://semver.org/). I think we'll likely have to do something like `1.0.0-1`.